### DISCUSSION TITLE
Easier upload mechanism from google photos album

### DISCUSSION THREAD

**Discussion by whoami730** [ente-io/ente#3794] [Enhancements]
Hi!
I am wondering if it is possible to figure out and improve the workflow of migrating a google album to an ente album? Currently, this is how I am doing it - download it on my local somewhere from google album and then upload it to a corresponding ente album. The issue here is that - 1. The local copy no doubts uses local space, and 2. Ente does not allow multiple uploads. So if I am migrating multiple albums, it becomes harder.

Here is the context for why I require this - 
Most (infact probably all) of my friends use Google Photos and therefore, whenever it comes to collaboration, they do create google photos album where everybody uploads the images they need. Now, I do not use Google Photos and therefore I prefer pulling in my photos from the album and adding them to Ente instead. If there could be a seamless way for uploading images from google photos (even better, a full album) directly to ente, without having to go through the takeout or download mechanism, it would make things much simpler.

Thanks!

**Comment by vishnukvmd** [comment]
Google unfortunately does not provide API access: https://ente.io/blog/google-takeout/

Could you please elaborate on
> 2. Ente does not allow multiple uploads

**Comment by whoami730** [comment]
> Could you please elaborate on 
> > Ente does not allow multiple uploads

You can't drop in multiple albums to Ente, because, once an upload is running, it doesn't allow uploading anything else till the ongoing one finishes.
